docs/en/latest/stand-alone.md

```diff
@@ -34,13 +34,15 @@ The routing rules in the `conf/apisix.yaml` file are loaded into memory immediat
 
 Since the current Admin API is based on the etcd configuration center solution, enable Admin API is not allowed when the Stand-alone mode is enabled.
 
-To enable Stand-alone mode, we can set `apisix.config_center` to `yaml` and disable Admin API in file `conf/config.yaml`.
+The Stand-alone mode can only be enabled when set the role of APISIX as data plane. We can set `deployment.role` to `data_plane` and `deployment.role_data_plane.config_provider` to `yaml`, and also need to disable Admin API in file `conf/config.yaml`.
```

Comment:
When we set `deployment.role` to `data_plane`, the admin API is disabled automatically.

Reply:
updated.